packages/components/Checkbox/src/Checkbox.tsx

```diff
@@ -48,22 +48,22 @@ export const Checkbox = compose<CheckboxType>({
       const checkmarkPath = thickCheckmark ? (
         <Path
           fill="currentColor"
-          d="M10.2637 1.26367L4 7.5332L0.736328 4.26367L1.26367 3.73633L4 6.4668L9.73633 0.736328L10.2637 1.26367Z"
+          d="M9.76497 3.20474C10.0661 3.48915 10.0797 3.96383 9.79526 4.26497L5.54526 8.76497C5.40613 8.91228 5.21332 8.99703 5.01071 8.99993C4.8081 9.00282 4.61295 8.92361 4.46967 8.78033L2.21967 6.53033C1.92678 6.23744 1.92678 5.76257 2.21967 5.46967C2.51256 5.17678 2.98744 5.17678 3.28033 5.46967L4.98463 7.17397L8.70474 3.23503C8.98915 2.9339 9.46383 2.92033 9.76497 3.20474Z"
```

Comment:
With this change we can remove the "thickCheckmark" logic, since we're now defaulting to a thicker checkmark.

Reply:
Removed the ternary operator in the newest commit.